x-pack/plugins/lens/public/app_plugin/lens_top_nav.tsx

```diff
@@ -570,12 +576,30 @@ export const LensTopNavMenu = ({
             getEsQueryConfig(uiSettings)
           );
 
-          return discover.locator!.getRedirectUrl({
+          let breakdownField;
```

Comment:
@stratoula Thanks for the feedback! I'll have to defer to Dima on this since it was an area where we split work for the PR. @dimaanj Is this something you're able to follow up on? I can look into it if necessary, just might take me a bit longer since I'm less familiar with the changes.

Reply:
Happy to help here if it is not clear!

Reply:
I think we should first decide how do we handle case with multiple breakdown fields seleccted, since Discover has only one.

Reply:
So, we ended up with decision not to provide any breakdown field from Lens to Discover. JYI @ninoslavmiskovic I will revert that part of the code as it was.